core/ledger/test/test.yaml

```diff
@@ -0,0 +1,17 @@
+###############################################################################
+#
+#    Peer section
+#
+###############################################################################
+peer:
+    # Path on the file system where peer will store data
+    fileSystemPath: /var/hyperledger/test/ledger_test
```

Comment:
I would like us to be able to move stuff around/refactor/clear temp files, not worry about name-, file-, dir- collisions, etc. `tmp` and `temp` is the new permanent! Easier also in terms of more consistent guidelines for new tests/etc...